src/client.c

```diff
@@ -317,10 +317,10 @@ static bool finish_set_pool(PgSocket *client, bool takeover)
 	struct HBARule *rule = NULL;
 
 	if (!client->login_user->mock_auth && !client->db->fake) {
-		PgUser *pool_user;
+		PgAuthInfo *pool_user;
```

Comment:
Here, for example - I would think pool_user could have a better name given its new type to reduce confusion.